indexer/node/header_traversal_test.go

```diff
@@ -55,25 +55,31 @@ func TestHeaderTraversalNextHeadersNoOp(t *testing.T) {
 }
 
 func TestHeaderTraversalNextHeadersCursored(t *testing.T) {
-	client := new(MockEthClient)
+	client := &testutils.MockClient{}
+	rpc := &testutils.MockRPC{}
+	client.Mock.On("RPC").Return(rpc)
```

Comment:
I think the mock-clients may need an `m.AssertExpectations(t)` at the end of the test.

Reply:
good call, wasn't aware this needed to be called, updated!